hw/hw06/series.py

```diff
@@ -0,0 +1,80 @@
+# HW6 Mathmatical Series
+
+
+def fibonacci(n):
+    """Defining a function  called fibonacci to provide nth value in fibonacci
+    sequence."""
+    if n == 0:
+        return 0
+    # First value in fibonacci sequence is 0.
+    elif n == 1:
+        return 1
+    # Second value in fibonacci sequence is 1
+    else:
+        return fibonacci(n-1) + fibonacci(n-2)
+    """Formula for fibonacci sequence after first two values.  Function calls
+    itself and sums the values to provide the nth value starting from 0."""
+
+
+def fib(n):
+    """Function to print fibonacci function"""
+    print (fibonacci(n))
+
+
+fib(3)  # test to see if fibonacci function is printing correct value
+fib(5)  # test to see if fibonacci function is printing correct value
+
+
+def lucas(n):
+    """New function called lucas to provide nth value in lucas numbers."""
+    if n == 0:
+        return 2
+    # Setting the first value to match that of the value in Lucas numbers.
+    elif n == 1:
+        return 1
+    # Setting the second value to match that of the value in Lucas numbers.
+    else:
+        return lucas(n-1) + lucas(n-2)
+    """Formula for Lucas numbers after first two values. Function calls itself and
+    sums the values to provide the nth value starting from 0."""
+
+
+def luc(n):
+    """Function to print lucas function."""
+    print (lucas(n))
```

Comment:
Print is a function in python 3. It's best to not have whitespace before the ().